docs/source/testing/mocking.md

```diff
@@ -73,9 +77,14 @@ server.listen().then(({ url }) => {
 });
 ```
 
-Similarly to `resolvers`, `mocks` allows the description of object types with the fields. Take note that the value corresponding to `Person` is a function that returns an object that contains fields pointing at more functions:
+You can also use `mocks` to define object types and the fields belonging to those object types (much like a [resolver map](../data/resolvers/#base-syntax)). 
+
+In the below example, take note that our `mocks`' `Person` object will invoke a function returning an object with fields pointing at more functions:
```

Comment:
"take note" -> "note"

"`mocks`' `Person`" -> "mocked `Person`"

"will invoke" -> "calls"

"more" -> "other" (and italicize, I think)